pageserver/src/tenant/remote_timeline_client/download.rs

```diff
@@ -229,6 +229,7 @@ async fn download_object(
                     || IoBufferMut::with_capacity(super::BUFFER_SIZE),
                     gate.enter().map_err(|_| DownloadError::Cancelled)?,
                     ctx,
+                    info_span!(parent: None, "download_object_buffered_writer", %dst_path),
```

Comment:
I haven't checked the ingest code, but are these writers kind of short-lived (like one wal record?)

Just thinking about perf impact of the extra span

Reply:
bench_ingest showed no impact https://github.com/neondatabase/neon/pull/10993#issuecomment-2690354492